python/lsst/daf/butler/core/s3utils.py

```diff
@@ -126,3 +129,38 @@ def bucketExists(bucketName, client=None):
         return True
     except s3.exceptions.NoSuchBucket:
         return False
+
+
+def setAwsEnvCredentials(accessKeyId='dummyAccessKeyId', secretAccessKey="dummySecretAccessKey"):
+    """Set AWS credentials environmental variables AWS_ACCESS_KEY_ID and
+    AWS_SECRET_ACCESS_KEY.
+
+    Parameters
+    ----------
+    accessKeyId : `str`
+        Value given to AWS_ACCESS_KEY_ID environmental variable. Defaults to
+        'dummyAccessKeyId'
+    secretAccessKey : `str`
+        Value given to AWS_SECRET_ACCESS_KEY environmental variable. Defaults
+        to 'dummySecretAccessKey'
+
+    Returns
+    -------
+    setEnvCredentials : `bool`
+        True when environmental variables were set, False otherwise.
+    """
+    if "AWS_ACCESS_KEY_ID" not in os.environ or "AWS_SECRET_ACCESS_KEY" not in os.environ:
```

Comment:
Maybe add a Notes section to the docs saying that you overwrite both values if any one value is not set.

Reply:
Added Notes section.